servers/display_server.h

```diff
@@ -70,6 +70,7 @@ class DisplayServer : public Object {
 		DISPLAY_HANDLE,
 		WINDOW_HANDLE,
 		WINDOW_VIEW,
+		OPENGL_CONTEXT,
```

Comment:
New constant should be bound here:
https://github.com/godotengine/godot/blob/9ff3a43a329182ba2fad34263297052c70a4524e/servers/display_server.cpp#L834-L837

Reply:
Ah, thanks for catching that! Added in my latest push